src/planner.c

```diff
@@ -777,7 +777,7 @@ timescaledb_set_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, Index rti, Rang
 					ts_cm_functions->set_rel_pathlist_dml(root, rel, rti, rte, ht);
 				break;
 			}
-			/* Fall through */
+			/* fall through */
 		default:
```

Comment:
This is the only change in the file and it does not seem important. Would suggest to remove it to keep the git history clean.